packages/synthetix-main/contracts/interfaces/IFundModule.sol

```diff
@@ -5,4 +5,27 @@ interface IFundModule {
     function initializeFundModule() external;
 
     function isFundModuleInitialized() external view returns (bool);
+
+    // function adjust(
```

Comment:
commented out to let it compile and not show lots of warnings. Will be completed in the Funds PR